src/pages/integrations/iterable/audience/index.md

```diff
@@ -28,7 +32,7 @@ From the List View, get the List ID for your List from the table to enter in the
 
 ## User Identity Mapping
 
-When forwarding audience data to Iterable, mParticle will send Emails and if they exist, Customer IDs and any additional user attributes. 
+When forwarding audience data to Iterable, mParticle will only send users with an email identity unless you are using MPID as the user identifier.
```

Comment:
The module registration response includes both email/customerid so both will always be sent to Iterable along with the mPid.   Should this mention more about the Iterable processing based on the User ID setting?

Reply:
That's a good point. I added a much more detailed description of how it works. Let me know what you think.